torch/_inductor/fx_passes/post_grad.py

```diff
@@ -396,3 +396,12 @@ def is_valid_splitwithsizes_cat(match):
 )
 def splitwithsizes_cat_replace(match, input_):
     return input_
+
+
+def view_to_reshape(gm):
```

Comment:
Maybe check that the outputs of the view are not mutated ?

Reply:
This is a good point, but we've already done functionalization in aot-autograd?

Reply:
We keep in the copy_ nodes in a mutation epilogue

Reply:
Create an issue to track this: https://github.com/pytorch/pytorch/issues/102229